frontend/app/src/components/home/ReplyingTo.svelte

```diff
@@ -48,8 +42,9 @@
     </h4>
     <ChatMessageContent
         {readonly}
-        {chatId}
+        messageContext={replyingTo.sourceContext}
```

Comment:
Again - I'm slightly worried that his might be unsafe. I'm assuming that `chatId` is the current chat, but the `replyingTo.sourceContext` is not necessarily the current chat. So I'm a bit worried this might be doing something subtly different now.